backend/app/libs/price_stats.py

```diff
@@ -96,9 +96,12 @@ def calculate_risk_contributions(
 
     summed_component_contributions = np.sum(component_contributions)
 
-    assert isclose(
-        summed_component_contributions, std_dev[0][0], rel_tol=0.0001
-    ), "error in calculations"
+    try:
```

Comment:
You're right, it was another bug I classified as being a "hotfix". This is introduced because of the Gnosis treasury outlier (`0x0DA0C3e52C977Ed3cBc641fF02DD271c3ED55aFe`).

I have a hunch this is caused by the small asset filter. I will open a separate PR shortly after removing this filter. We can move discussions over to that PR.